owncloudApp/src/main/res/layout/preview_image_fragment.xml

```diff
@@ -16,51 +16,88 @@
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
   
 -->
-
-<!--
-     ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/fdScrollView"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent"
-    android:background="@color/background_color" 
-    android:gravity="center_horizontal"
--->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/top"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="#000000"
-    tools:context=".ui.preview.PreviewImageFragment"
     android:filterTouchesWhenObscured="true"
-    >
+    tools:context=".ui.preview.PreviewImageFragment">
 
     <ProgressBar
         android:id="@+id/progressWheel"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_centerInParent="true"
```

Comment:
Some attributes were only useful in RelativeLayout. If we update it to constraint, we could remove some of them. For example, this one. Don't u think?